templates/compose/teamspeak.yaml

```
@@ -0,0 +1,40 @@
+  mariadb:
+    image: 'mariadb:latest'
+    volumes:
+      -
+        type: bind
+        source: ./teamspeak-db
+        target: /var/lib/mysql
+        is_directory: true
+    restart: always
+    environment:
+      MYSQL_ROOT_PASSWORD: '${SERVICE_PASSWORD_DB}'
+      MYSQL_DATABASE: teamspeak
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)